packages/chain-adapters/src/ethereum/EthereumChainAdapter.ts

```diff
@@ -277,9 +277,9 @@ export class ChainAdapter implements IChainAdapter<ChainTypes.Ethereum> {
 
     const feeData = (await this.providers.http.getGasFees()).data
     const normalizationConstants = {
-      instant: String(new BigNumber(fees.instant).dividedBy(fees.fast)),
+      instant: new BigNumber(fees.instant).dividedBy(fees.fast).toString(),
```

Comment:
we have a utility called `bnOrZero` that we use everywhere, it just safely wraps `new BigNumber()` to stop runtime errors.

can you please change all the cases of `new BigNumber()` (on lines that you have touched) to use `bnOrZero`?

Reply:
Yes, I can do that.